docs/source/release/v4.0.0/indirect_inelastic.rst

```diff
@@ -163,9 +163,8 @@ Bugfixes
   from their default.
 - A bug where the output reduced files had large file sizes depending on the size of the batch
   being reduced from the :ref:`ISISIndirectEnergyTransfer <algm-ISISIndirectEnergyTransfer>`
-  algorithm on the ISISEnergyTransfer interface has been fixed. The
-  :ref:`ISISIndirectEnergyTransferWrapper <algm-ISISIndirectEnergyTransferWrapper>` algorithm
-  should be used instead in order to avoid these large file sizes.
+  algorithm on the ISISEnergyTransfer interface has been fixed. The ``ISISIndirectEnergyTransferWrapper``
```

Comment:
Shouldn't this be mentioned as ISISIndirectEnergyTransfer?

Reply:
Because this is in the release notes for v4.0 we want to keep this the same as it is still an accurate log of what changed for v4.0! I needed to remove the link because the documentation page for `ISISIndirectEnergyTransferWrapper` will no longer exist when this PR is merged